src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.stories.tsx

```diff
@@ -327,7 +343,39 @@ export const DashboardPremiumUser: Story = {
           countryCode="us"
           user={userData}
           userBreaches={breachItemArraySample}
-          userScannedResults={scannedResultsArraySample}
+          userScanData={{
+            scan: scanSample,
+            results: scannedResultsArraySample,
+          }}
+          isEligibleForFreeScan={true}
+          locale={"en"}
+          bannerData={dashboardSummaryWithScan}
+          featureFlagsEnabled={{
+            FreeBrokerScan: true,
+            PremiumBrokerRemoval: true,
+          }}
+        />
+      </Shell>
+    );
+  },
+};
+
+export const DashboardPremiumUserAllResolved: Story = {
```

Comment:
This use case (a Premium has that has resolved everything) has its own top banner content, hence a new story (and test).